pyscript.core/src/plugins/py-editor.js

```diff
@@ -163,8 +166,18 @@ const init = async (script, type, interpreter) => {
 
     let isSetup = script.hasAttribute("setup");
     const hasConfig = script.hasAttribute("config");
+    const serviceWorker = script.getAttribute("service-worker");
     const env = `${interpreter}-${script.getAttribute("env") || getID(type)}`;
 
+    // helps preventing too lazy ServiceWorker initialization on button run
+    if (serviceWorker) {
+        const bootstrap = document.createElement("script");
+        bootstrap.type = "mpy";
```

Comment:
the sooner it bootstraps, the better ... arguably we could introduce a dummy interpreter *or* I could orchestrate a way more convoluted check via Service Worker registry and yada yada but that's really convoluted and error-prone by design so I prefer to confine that part in [sabayon](https://github.com/WebReflection/sabayon/blob/main/src/main.js#L70-L82) or we risk to race reloads and that'd be not cool at all.

Reply:
Aha... I did wonder. Does it need to be `mpy`... could we have a `js`..? Zero download etc... :stuck_out_tongue:

Reply:
it has to be recognized by polyscript .... we don't have a `js` interpreter so that cannot work, we could have a dummy module though, it will be handy in a way or another in the future, imho.

Reply:
MR here: this makes sense to me and it'll be handy in the future https://github.com/pyscript/polyscript/pull/110

Reply:
@ntoll I went ahead and published `dummy` interpreter in *polyscript* so right now this is the `type` used to bootstrap ASAP the Service Worker whenever provided and there is zero runtime/network cost involved.

If that's OK and you are happy now I can merge both *polyscript* MR and this one too.

Reply:
Also ... the code is (imho) much better now:

```js

    // helps preventing too lazy ServiceWorker initialization on button run
    if (serviceWorker) {
        new XWorker(
            'data:application/javascript,',
            { type: 'dummy', serviceWorker },
        );
    }
```

Reply:
Totally agreed..! :rocket: